test/unit/test_couchdb_session_auth.py

```diff
@@ -31,11 +31,15 @@
 request_kwargs = None
 
 
-def requests_request_spy(*args, **kwargs):
```

Comment:
Found some unused import statements:
 `import responses`
`from unittest import mock`

Reply:
Good call! I'll push a commit for this soon.

Reply:
Done in 4fb655b

Reply:
https://travis-ci.com/github/IBM/cloudant-python-sdk/jobs/479659186#L323 says `responses` is not unused :/

Reply:
Re-added in 2b139034092403a86fd55d70022095425c476907

Reply:
> https://travis-ci.com/github/IBM/cloudant-python-sdk/jobs/479659186#L323 says responses is not unused :/

Sorry, missed that.